doc/jvm/xgboost4j_spark_gpu_tutorial.rst

```diff
@@ -149,9 +149,8 @@ you can do it through setters in XGBoostClassifer:
 
 .. note::
 
-  In contrast to the XGBoost4J-Spark package, which needs to first assemble the numeric
-  feature columns into one column with VectorUDF type by VectorAssembler, the
-  XGBoost4J-Spark-GPU does not require such transformation, it accepts an array of feature
+  In different with XGBoost4j-Spark which accepts both a feature column with VectorUDT type and
+  an array of feature column names, XGBoost4j-Spark-GPU only accepts an array of feature
```

Comment:
I think the original wording was better.

Reply:
@sameerz, right now, we have unified the setFeaturesCol(value: Array[String]) API for both CPU and GPU. So users do not need to do the features assembling for xgboost4j-spark.